### ISSUE TITLE
PGP Sign

### ISSUE THREAD

**Issue by mickaelistria** [redhat-developer/eclipseide-jdtls#84]
Sign with PGP.

**Comment by mickaelistria** [comment]
I got in touch with some security experts at Red Hat and we agreed the simplest and safest approach is to create a new key for it.
So I've created a PGP key to sign eclipseide-jdtls: https://keyserver.ubuntu.com/pks/lookup?search=b65a772a26c6cfcd2abeb352accf6ef34a59ab7c&fingerprint=on&op=index
Then I'm setting up the GitHub secrets and the build to use this key to sign artifacts.
I've requested Red Hat security team to sign the key so it gets part of the Web Of Trust from Red Hat (using the `redhatdevel` key for affiliated development work that are not covered by Red Hat product SLA) https://issues.redhat.com/browse/SIGNSERVER-756
But I believe we;ll face https://github.com/eclipse-tycho/tycho/issues/2477 and will need to fix Tycho for completion here.
